scripts/validate

```diff
@@ -31,5 +31,5 @@ go mod tidy
 if [ -n "$DIRTY" ]; then
     echo Source dir is dirty
     git status --porcelain --untracked-files=no
-    exit 1
+    #exit 1
```

Comment:
Is this meant to be here or some debugging accidentally left in or what?
Should we just delete the line all together?